game.js

```diff
@@ -10,3 +10,16 @@ const endGame = () => {
   const customWindow = window.open('', '_blank', '');
   customWindow.close();
 }
+
+
+sscore=0;
+
+
+function printscore(j){/*game ver.commit 1.js에서 score를 받아서 print하는 함수*/
+  sscore=localStorage.getItem('score')
+  j.text(sscore);
+}
+
+$(document).ready(function(){/*점수를 print*/
+  printscore($("#score"));
+});
```

Comment:
end.html 파일에서 body 하단부에 script src를 지정해줬기 때문에 15~25번째 라인에 해당되는 부분은 아래 한 줄로 줄여서 쓸 수 있습니다.

```suggestion
$("#score").text(localStorage.getItem('score'));
```